src/plays/photoshopApp/Photoshopapp.js

```diff
@@ -0,0 +1,140 @@
+import PlayHeader from 'common/playlists/PlayHeader';
+import './styles.css';
+import React, { useState } from 'react';
+// import './App.css';
+import Slider from './Slider';
+import SidebarItem from './SidebarItem';
+// WARNING: Do not change the entry componenet name
+
+const DEFAULT_OPTIONS = [
+  {
+    name: 'Brightness',
+    property: 'brightness',
+    value: 100,
+    range: {
+      min: 0,
+      max: 200
+    },
+    unit: '%'
+  },
+  {
+    name: 'Contrast',
+    property: 'contrast',
+    value: 100,
+    range: {
+      min: 0,
+      max: 200
+    },
+    unit: '%'
+  },
+  {
+    name: 'Saturation',
+    property: 'saturate',
+    value: 100,
+    range: {
+      min: 0,
+      max: 200
+    },
+    unit: '%'
+  },
+  {
+    name: 'Grayscale',
+    property: 'grayscale',
+    value: 0,
+    range: {
+      min: 0,
+      max: 100
+    },
+    unit: '%'
+  },
+  {
+    name: 'Sepia',
+    property: 'sepia',
+    value: 0,
+    range: {
+      min: 0,
+      max: 100
+    },
+    unit: '%'
+  },
+  {
+    name: 'Hue Rotate',
+    property: 'hue-rotate',
+    value: 0,
+    range: {
+      min: 0,
+      max: 360
+    },
+    unit: 'deg'
+  },
+  {
+    name: 'Blur',
+    property: 'blur',
+    value: 0,
+    range: {
+      min: 0,
+      max: 20
+    },
+    unit: 'px'
+  }
+];
+
+function Photoshopapp(props) {
+  // Your Code Start below.
+  const [selectedOptionIndex, setSelectedOptionIndex] = useState(0);
+  const [options, setOptions] = useState(DEFAULT_OPTIONS);
+  const selectedOption = options[selectedOptionIndex];
```

Comment:
Here we are deriving the value from the state, so I would recommend using the `useMemo` hook here so that `selectedOption` have the updated value based on dependent variables.

```suggestion
  const selectedOption = useMemo(()=>options[selectedOptionIndex],[options,selectedOptionIndex]);
```